arm-software/embedded/test/multilib/armv6m.test

```diff
@@ -1,13 +1,13 @@
 # RUN: %clang -print-multi-directory --target=armv6m-none-eabi -mfpu=none | FileCheck %s --check-prefix=CHECK-NOFP-EXN-RTTI-UNALIGNED
```

Comment:
Change CHECK-NO-FP-EXN-RTTI-UNALIGNED to CHECK-NO-FP-EXN-RTTI. That is, remove the "-UNALIGNED" suffix.